dao/reminders.js

```diff
@@ -139,8 +140,9 @@ module.exports = {
     debug('findAllDueReminders(now=%d)', now);
     return database.ready.then(db =>
       db.all(
-        'SELECT * FROM reminders WHERE due <= ? AND status = "waiting"',
-        now
+        'SELECT * FROM reminders WHERE due <= ? AND status = ?',
+        now,
+        Status.WAITING.toString()
```

Comment:
I'm not a huge fan of calling explicitly `toString()`, here. If we use JS template strings, we don't require this call. See [this example](https://github.com/fxbox/calendar-server/pull/31/commits/180bf27aefb0f3c840c1ded5a61b0c34333b76a6#diff-590956fbafee872e143f12226c7915fbR130)